c2corg_ui/templates/base.html

```diff
@@ -149,6 +151,10 @@
            vcRecaptchaServiceProvider.setSiteKey('${settings["recaptcha_site_key"]}');
          });
 
+         angular.module('app').constant('angularMomentConfig', ['moment-timezone'], {
+           timezone: 'Europe/London'
```

Comment:
According to [SO comment](http://stackoverflow.com/a/25284017/4114514), it is possible to just set it as `value `instead of `constant `and it should be handled automatically. Later it is probably not needed to set this part at all, but I would leave it here now until we fix the issue.